dot-net-standalone/README.md

```diff
@@ -0,0 +1,41 @@
+# .Net with Nx
+
+An example repository showing how to use [Nx](https://nx.dev) with [.Net](https://learn.microsoft.com/en-us/aspnet/core/?view=aspnetcore-7.0) projects using [@nx/dotnet](https://www.nx-dotnet.com/).
+
+## What's inside
+ This repository contains
+ - A .Net Web Api project
+ - A .Net class library
+ - A .Net test project
+
+ ## How to run it
+
+ ### Build
+ To build the application run the following command:
+ ```shell
+ npx nx build api
```

Comment:
just running `npx nx build` should be enough, right?

Reply:
There are two projects generated `api` and `testing`  I guess I could just use one `project.json` and change the cwd to `e2e` for testing